dideo.owl

```diff
@@ -11499,10 +11502,10 @@ A cultured cell population maintained in vitro: &quot;Rat cortical neurons from
         <rdf:type rdf:resource="http://purl.obolibrary.org/obo/OBI_0302900"/>
     </owl:NamedIndividual>
     <rdf:Description>
-        <rdfs:comment rdf:datatype="http://www.w3.org/2001/XMLSchema#boolean">true</rdfs:comment>
+        <owl:deprecated></owl:deprecated>
     </rdf:Description>
     <rdf:Description>
-        <owl:deprecated></owl:deprecated>
+        <rdfs:comment rdf:datatype="http://www.w3.org/2001/XMLSchema#boolean">true</rdfs:comment>
```

Comment:
Another automatic change by the Protege Desktop IDE.  This could be removed or split out into a separate commit.